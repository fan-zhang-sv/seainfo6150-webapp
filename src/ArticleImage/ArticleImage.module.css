.image {
  border: 10px solid #c00;
  border-radius: 50%;
  width: 100%;
  margin-bottom: 20px;
  box-shadow: 0 8px 8px gray;
}

@media screen and (min-width: 525px) {
  .image {
    width: 75%;
  }
}

@media screen and (min-width: 800px) {
  .image {
    width: 60%;
  }
}

@media screen and (min-width: 1150px) {
  .image {
    width: 55%;
  }
}
