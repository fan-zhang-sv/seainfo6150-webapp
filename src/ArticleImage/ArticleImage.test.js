import React from "react";
import ArticleImage from "./ArticleImage";

describe("ArticleImage tests", () => {

  it("renders correctly", () => {
    const { container } = render(<ArticleImage url="https://images.unsplash.com/photo-1606015218098-77835ec98075?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=2134&q=80" title="test" />);
    expect(container).toMatchSnapshot();
  });

});