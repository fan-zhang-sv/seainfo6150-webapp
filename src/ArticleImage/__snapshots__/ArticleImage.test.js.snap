// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ArticleImage tests renders correctly 1`] = `
<div>
  <img
    alt="test"
    class="image"
    src="https://images.unsplash.com/photo-1606015218098-77835ec98075?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=2134&q=80"
  />
</div>
`;
